lib/spack/spack/schema/projections.py

```diff
@@ -14,7 +14,6 @@
 properties = {
     'projections': {
         'type': 'object',
-        'default': {},
```

Comment:
(question) what does removing this default do?

Reply:
It makes Spack not write out the empty projection any time you update the root through the command line API (`spack config add config:install_tree:root:/path/to/root`)